src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -255,5 +255,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return DirectCast(n.GetSyntax(cancellationToken), TypeStatementSyntax).Modifiers.Any(SyntaxKind.PartialKeyword)
                 End Function)
         End Function
+
+        Public Function IsPossibleTupleContext(semanticModel As SemanticModel, position As Integer, cancellationToken As CancellationToken) As Boolean Implements ISemanticFactsService.IsPossibleTupleContext
+            Throw New NotImplementedException()
```

Comment:
hrmm.. why is the VB impl unused?  Don't we want to have the same sort of completion logic for VB tuples?